compiler/lineinfos.nim

```diff
@@ -32,13 +32,13 @@ type
     # non-fatal errors
     errIllFormedAstX, errCannotOpenFile,
     errXExpected,
-    errGridTableNotImplemented,
+    errRstGridTableNotImplemented,
     errMarkdownIllformedTable,
```

Comment:
move up errMarkdownIllformedTable 1 spot? ABI compat isn't mandated

Reply:
why? this order is the same in all the 3 files.

Reply:
to avoid splitting the `errRstxxx` entries with `errMarkdownIllformedTable` in the middle

Reply:
changed to `errRstMarkdownIllFormadTable` for uniformity.